Source/Eto.WinForms/Drawing/GraphicsHandler.cs

```diff
@@ -62,19 +62,24 @@ public GraphicsHandler()
 
 		bool shouldDisposeGraphics = true;
 
-		public GraphicsHandler(sd.Graphics graphics, bool shouldDisposeGraphics = true)
+		/// <summary>
+		/// Create handler for <see cref="Graphics.IHandler"/>.
+		/// </summary>
+		/// <param name="graphics">Native graphics to use</param>
+		/// <param name="dispose">Pass ownership and dispose native graphics in Dispos></param>
+		public GraphicsHandler(sd.Graphics graphics, bool dispose)
```

Comment:
Refactoring and description, because converting disposable to another disposable is dangerous. We agreed to state ownership explicitly (non-default `bool dispose`).